test/unit/app/lib/send-email.test.js

```diff
@@ -44,24 +58,67 @@ describe('Send email test', () => {
+    sendMessage.mockResolvedValueOnce(true)
+    await sendEmail.sendFarmerConfirmationEmail({ reference, sbi, whichSpecies, startDate, userType, email, farmerName, orgData })
+    expect(sendMessage).toHaveBeenCalledTimes(1)
+    expect(sendMessage).toHaveBeenCalledWith({ reference, sbi, whichSpecies, startDate, userType, email, farmerName, name: orgData.orgName, orgEmail }, applicationEmailDocRequestMsgType, applicationdDocCreationRequestQueue)
+  })
 
+  test('sendFarmerClaimConfirmationEmail returns true on successful email', async () => {
+    notifyClient.sendEmail.mockResolvedValueOnce(error)
     const response = await sendEmail.sendFarmerClaimConfirmationEmail(email, reference)
-
     expect(notifyClient.sendEmail).toHaveBeenCalledWith(templateIdFarmerClaimComplete, email, { personalisation: { reference }, reference })
     expect(response).toBeTruthy()
   })
 
+  test('sendFarmerClaimConfirmationEmail returns true on successful email via SFD', async () => {
+    conf.sfdMessage.enabled = true
+    sendSFDEmail.mockResolvedValueOnce(error)
+    const response = await sendEmail.sendFarmerClaimConfirmationEmail(email, reference)
+    expect(sendSFDEmail).toHaveBeenCalledWith(templateIdFarmerClaimComplete, email, { personalisation: { reference }, reference })
+    expect(response).toBeTruthy()
+  })
+
   test('sendFarmerClaimConfirmationEmail returns false on error sending email', async () => {
     notifyClient.sendEmail.mockRejectedValueOnce(error)
-
     const response = await sendEmail.sendFarmerClaimConfirmationEmail(email, reference)
+    expect(response).toBe(false)
+  })
 
+  test('sendFarmerClaimConfirmationEmail returns false on error sending email via SFD', async () => {
+    conf.sfdMessage.enabled = true
+    sendSFDEmail.mockRejectedValueOnce(error)
+    const response = await sendEmail.sendFarmerClaimConfirmationEmail(email, reference)
     expect(response).toBe(false)
   })
+
   describe('sendFarmerEndemicsClaimConfirmationEmail', () => {
     test('sendFarmerEndemicsClaimConfirmationEmail sends email to farmer email', async () => {
+      const data = {
+        reference: 'RESH-DFEF-6037',
+        applicationReference: 'AHWR-B977-4D0D',
+        amount: '£[amount]',
+        orgData: {
+          orgEmail: 'test@unit-test.org'
+        }
+      }
+      const templateId = 'templateIdFarmerEndemicsClaimComplete'
+      const expectedPersonalisation = {
+        reference: data.reference,
+        applicationReference: data.applicationReference,
+        amount: data.amount || '£[amount]'
+      }
+
+      const result = await sendEmail.sendFarmerEndemicsClaimConfirmationEmail(data, templateId)
+      expect(result).toBe(true)
+      expect(notifyClient.sendEmail).toHaveBeenCalledWith(templateId, data.orgData.orgEmail, { personalisation: expectedPersonalisation, reference: data.reference })
+    })
+
+    test('sendFarmerEndemicsClaimConfirmationEmail sends email to farmer email via SFD', async () => {
```

Comment:
This test is not testing what it states it will, 'sends email to farmer email via SFD' is asserting an email is sent to the org email address.
Might be better to restructure these tests a bit such that you combine them a bit and set up your test fixture/input data, and then check it is sent to each of the email addresses as expected, rather than having a test per email address